ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -580,6 +581,41 @@ def power(x1, x2, /):
     return ivy.pow(x1, x2)
 
 
+@with_supported_dtypes(
+    {"2.5.1 and below": ("float32", "float64", "int32", "int64")}, "paddle"
```

Comment:
Since this is the jax frontend, only jax unsupported dtypes should be included here. For paddle, it will be handled in the paddle frontend.

Reply:
Hey @bipinKrishnan - All done. Please review if this looks fine.

Reply:
@bipinKrishnan - Can we merge and close this if it looks fine now?